newIDE/app/src/MainFrame/index.js

```diff
@@ -1130,9 +1179,11 @@ class MainFrame extends React.Component<Props, State> {
     this.setState(
       {
         editorTabs: openEditorTab(this.state.editorTabs, {
-          name: name + ' ' + i18n._(t`(Extension)`),
+          label: name + ' ' + i18n._(t`(Extension)`),
           renderEditor: ({ isActive, editorRef }) => (
             <EventsFunctionsExtensionEditor
+              onPreview={() => {}}
```

Comment:
You're sure about this? 🧐
Better pass null than an empty function. Because `null` is explictly saying "there is no way to run a preview, don't even try to call me". While passing an empty function is "hacky". It's like saying "See I passed you something to run the preview, so feel free to show a button... but well actually I won't do anything haha! Oops sorry."

Reply:
Haha ok, i like when the values talk :+1: